samples/matrix_mul/matrix_mul.cpp

```diff
@@ -38,6 +38,7 @@ int main() {
       int j = idx.get_global_id(1);
 
       c_acc[i][j] = 0.0f;
+      for (int _ = 0; _ < 100; _++)
```

Comment:
We could keep clear the code samples from testing stuff. Maybe let's remove this or limit its execution to some testing flags.

Reply:
Okay, I'm gonna remove the loop